fitlins/interfaces/bids.py

```diff
@@ -157,14 +157,14 @@ def _run_interface(self, runtime):
         selectors = self.inputs.selectors
 
         analysis = ba.Analysis(model=self.inputs.model, layout=layout)
-        analysis.setup(**selectors)
+        analysis.setup(drop_na=False, **selectors)
         block = analysis.blocks[0]
 
         entities = []
         session_info = []
         contrast_info = []
         for paradigm, _, ents in block.get_design_matrix(
-                block.model['HRF_variables'], mode='sparse'):
+                block.model['HRF_variables'], mode='sparse', force=True):
```

Comment:
By default, you only get back variables that or of the specified type (`'sparse'` or `'dense'`), and others are ignored. The `force` argument controls whether variables of the other type are forced to the target representation (i.e., dense to sparse, or sparse to dense).

Reply:
Sparsi-fying dense events. This is case that an HRF_variable was densified in place. This is necessary for certain operation, such as orthogonality. If nistats allowed dense events to be convolved, this would not be necessary.

Reply:
I see. Does this work well? I would imagine any orthogonalization that happens pre-convolution would not make something that can be nicely convolved...

Reply:
If the orthogonalization is supposed to happen in psychological space, then it should be done pre-convolution.